jparsec-examples/src/test/java/org/codehaus/jparsec/examples/statement/FirstTest.java

```diff
@@ -0,0 +1,97 @@
+package org.codehaus.jparsec.examples.statement;
+
+import static org.junit.Assert.assertEquals;
+
+import org.codehaus.jparsec.Parser;
+import org.codehaus.jparsec.Parsers;
+import org.codehaus.jparsec.Scanners;
+import org.codehaus.jparsec.Terminals;
+import org.codehaus.jparsec.examples.statement.ast.DoubleExpression;
+import org.codehaus.jparsec.examples.statement.ast.SingleExpression;
+import org.junit.Test;
+
+public class FirstTest {
+
+	//low-level grammar
+	private static final Terminals OPERATORS =
+			Terminals.operators("=", "readonly", "var");
+
+	//for integers and identifiers we need a PARSER and a TOKENIZER for each
+	//here are the parsers
+	public static Parser<String> identSyntacticParser = Terminals.Identifier.PARSER;
+	public static Parser<String> integerSyntacticParser = Terminals.IntegerLiteral.PARSER;
+	
+	//and here are the tokenizers, combined into our single tokenizer
+	static final Parser<?> TOKENIZER = Parsers.or(
+					OPERATORS.tokenizer(),
```

Comment:
Indentations don't seem consistent in this file. How about we always indent by 4 spaces?